src/core/ledger/transaction.ts

```diff
@@ -228,7 +228,7 @@ export const addOnUsPayment = async ({
     })
     .debit(payerUser.accountPath, sats * payerUser.ratioBtc, {
       ...metadata,
-      memoPayer,
+      memoPayer: memoPayer,
```

Comment:
I dont understand this change

Reply:
The result of one commit changing it to `memoPayer: memoFromPayer` and another changing it back via a search-and-replace 😂